x/ibc/core/03-connection/keeper/keeper.go

```diff
@@ -45,6 +45,16 @@ func (k Keeper) GetCommitmentPrefix() exported.Prefix {
 	return commitmenttypes.NewMerklePrefix([]byte(k.storeKey.Name()))
 }
 
+// GenerateConnectionIdentifier returns the next connection identifier.
+func (k Keeper) GenerateConnectionIdentifier(ctx sdk.Context) string {
+	nextConnSeq := k.GetNextConnectionSequence(ctx)
+	connectionID := fmt.Sprintf("%s%d", types.ConnectionPrefix, nextConnSeq)
```

Comment:
should we add a separator here to avoid collisions?

Reply:
Collisions with what? The identifier is auto derived, not user inputted